src/User/src/Entity/UserDetail.php

```diff
@@ -36,10 +36,10 @@ class UserDetail extends AbstractEntity
     protected ?string $lastName = null;
 
     /**
-     * @ORM\Column(name="email", type="string", length=191, nullable=true)
-     * @var string|null $email
+     * @ORM\Column(name="email", type="string", length=191, nullable=false)
```

Comment:
@OnitaAndrei a column is by default not nullable, you can remove `nullable=false` .